DeviceDiagnostics/DeviceDiagnostics.cpp

```diff
@@ -71,8 +71,6 @@ namespace WPEFramework
         /* virtual */ const string DeviceDiagnostics::Initialize(PluginHost::IShell* service)
         {
 #ifdef ENABLE_ERM
-            int ret;
-
```

Comment:
please revert

Reply:
This variable is never used, I just make cleanup in code